x-pack/docs/en/rest-api/security/create-role-mappings.asciidoc

```diff
@@ -77,10 +77,14 @@ _Exactly one of `roles` or `role_templates` must be specified_.
 `rules`::
 (Required, object) The rules that determine which users should be matched by the
 mapping. A rule is a logical condition that is expressed by using a JSON DSL.
-See  <<role-mapping-resources>>. 
+See  <<role-mapping-resources>>.
 
 ==== Role Templates
 
+NOTE: Role templates require the relevant scripting feature to be enabled,
```

Comment:
I think this NOTE should move below the introductory paragraph, since at this point we haven't defined role templates yet.

Reply:
Thanks Lisa. Please make changes as you see fit. Thanks a lot.

Reply:
Done, thanks!